xtask/src/main.rs

```diff
@@ -21,6 +21,8 @@ struct Xtask {
 
 #[derive(Debug, StructOpt)]
 pub enum Command {
+    /// Locally run all the checks the CI will perform.
+    All,
```

Comment:
Do you know [`flatten`](https://docs.rs/structopt/0.3.22/structopt/#flattening)?

You won't even need to implement `Default` anymore :sunglasses: 

```rust
#[derive(Debug, StructOpt)]
pub struct All {
    #[structopt(flatten)]
    test: commands::Test,
    
    // you can even add the other commands here
}
```

Reply:
Oh this is cool! 🎉